lib/postmark/api_client.rb

```diff
@@ -258,13 +264,10 @@ def get_stats_totals(options = {})
 
     def get_stats_counts(stat, options = {})
       url = "stats/outbound/#{stat}"
-
```

Comment:
I updated get_triggers method in same file to support inbound rules retrieval, so I figured to clean new lines in get_stats_counts method too, since I am editing same file anyways and spotted it looking differently from other lines close to it. 

I can remove deleting emtpy lines though, but since I am editing file anyways, I think clearer view of that method without extra newlines is just beneficial.